platform/Dockerfiles/rust-hypervisor-firmware/Dockerfile

```diff
@@ -50,26 +50,18 @@ RUN git clone --depth 1 -b ${RUST_HYPERVISOR_FIRMWARE_VERSION} https://github.co
 WORKDIR /workspace/install
 RUN curl --location --remote-name https://static.rust-lang.org/rustup/dist/${RUST_ARCH}-unknown-linux-gnu/rustup-init
 RUN chmod +x rustup-init
-RUN ./rustup-init -y --profile minimal --default-host ${RUST_ARCH}-unknown-linux-gnu --default-toolchain stable --target ${RUST_ARCH}-unknown-none
-RUN cargo install cargo-deb --locked
-
-#RUN rustup component add rust-std
-#RUN rustup component add rust-src
-
 
-
-#RUN rustup default nightly
-#RUN rustup component add rust-src --toolchain nightly
+RUN ./rustup-init -y --profile minimal --default-host ${RUST_ARCH}-unknown-linux-gnu --default-toolchain nightly --target ${RUST_ARCH}-unknown-none
+RUN cargo install cargo-deb --locked
 
 
 ###
 #
 # Build rust-hypervisor-firmware
 
-WORKDIR /workspace/build/rust-hypervisor-firmware-${RUST_HYPERVISOR_FIRMWARE_VERSION}
-#RUN rustup target add ${RUST_ARCH}-unknown-none
+WORKDIR /workspace/build
+RUN rustup target add ${RUST_ARCH}-unknown-none
 RUN cargo build --release --target ${RUST_ARCH}-unknown-none.json -Z build-std=core,alloc -Z build-std-features=compiler-builtins-mem
-RUN ls -lR /workspace/build/rust-hypervisor-firmware-0.4.2/
 RUN cargo deb --no-build --no-strip --target ${RUST_ARCH}-unknown-none
 RUN find / -name \*deb -ls
```

Comment:
For some reason, this line now causes an error. I think it's debugging code which can be removed.